review_practice/react-app/src/pages/Login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import { useMutation } from '@apollo/react-hooks';
+
+import * as Style from './style';
+
+import * as AuthAPI from 'graphql/mutation/auth';
+
+import Logo from 'components/Logo';
+
+import { getToken, setToken } from 'service/TokenManager';
+
+const Login = ({ history }) => {
+    if (getToken()) history.push('/main');
+
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [joinUser, { data, loading, error }] = useMutation(AuthAPI.JOIN_USER);
+
+    useEffect(() => {
+        if (data) {
+            setToken(data.joinUser);
+            history.push('/main');
+        }
+        if (loading) console.log('로그인 중');
+        if (error) alert('입력란을 다시 한번 확인해주세요');
+    }, [data, loading, error]);
+
+    const join = () => joinUser({ variables: { email, password } });
+
+    return (
+        <Style.Main>
+            <Logo />
+            <Style.InputField>
+                <input id="email" type="text" placeholder="Email" onChange={e => setEmail(e.target.value)} />
```

Comment:
```suggestion
                <input id="email" type="email" placeholder="Email" onChange={e => setEmail(e.target.value)} />
```

P3; input type email을 사용할 수 있습니다. 좀더 많은 정보는 [mdn](https://developer.mozilla.org/ko/docs/Web/HTML/Element/Input)문서를 참고해보셔도 좋을것 같습니다!